engines/alpinefaction/assets/setup-alpine-url.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+SCRIPTPATH="$(readlink -f "$0")"
+SCRIPTDIR="$(dirname "$SCRIPTPATH")"
+read -r -n 1 -p 'This script will install URI handlers for af: and rf: URLs for Red Faction, are you ok with this? [y/N] ' _are_we_ok
```

Comment:
This won't be visible for someone running the game on steam, so what is this supposed to be doing?

Reply:
Ok I think once this is made as an otional script, if it's even possible, then this'll make sense. thanks!